.github/workflows/dailyWorkflow.yml

```diff
@@ -47,36 +47,21 @@ jobs:
           uses: ruby/setup-ruby@v1
           with:
             ruby-version: '3.2'
-        - name: Install dependencies
+        - name: Generate report
           run: |
-            python -m pip install --upgrade pip
-            pip install -r requirements.txt
-        - name: Run buildfarm scripts
+            cd database/scripts
+            echo "REPORT_NAME=$(./generate_report.rb)" >> $GITHUB_ENV
```

Comment:
Technically it should be `GENERATED_REPORT` or `REPORT_RESULT`, otherwise it looks like you are calculating the name first